src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+  --pf-c-sequence__link--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-sequence__link--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-sequence__link--BorderBottomColor: var(--pf-global--Color--200);
+  --pf-c-sequence__link--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__link--hover--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--active--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--focus--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // Number
+  --pf-c-sequence__number--Padding: var(--pf-global--spacer--xs);
+  --pf-c-sequence__number--MarginRight: var(--pf-global--spacer--lg);
+  --pf-c-sequence__number--MinWidth: var(--pf-global--spacer--lg);
+  --pf-c-sequence__number--MaxHeight: var(--pf-global--spacer--lg);
+  --pf-c-sequence__number--hover--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-sequence__number--hover--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--hover--MaxHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--active--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-sequence__number--active--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--active--MaxHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--focus--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-sequence__number--focus--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--focus--MaxHeight: var(--pf-global--spacer--xl);
+
+  // Number Color
+  --pf-c-sequence__number--Color: var(--pf-global--Color--200);
+  --pf-c-sequence__number--hover--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--active--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--focus--Color: var(--pf-global--primary-color--100);
+
+  // Number Font
+  --pf-c-sequence__number--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-sequence__number--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-sequence__number--hover--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__number--active--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__number--focus--FontSize: var(--pf-global--FontSize--xl);
+
+  // Number Border
+  --pf-c-sequence__number--BorderColor: var(--pf-global--Color--200);
+  --pf-c-sequence__number--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-sequence__number--BorderWeight: var(--pf-global--BorderWidth--md);
+  --pf-c-sequence__number--hover--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--active--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--focus--BorderColor: var(--pf-global--primary-color--100);
+
+  // Content
+  --pf-c-sequence__content--MinHeight: var(--pf-global--spacer--xl);
+
+  // content Title
+  --pf-c-sequence__content--title--MaxWidth: 100px;
```

Comment:
Can you change this to `#{pf-size-prem(100px)};`?

Reply:
yes changed this